test/Api.IntegrationTest/Controllers/SecretsControllerTests.cs

```diff
@@ -25,14 +26,91 @@ public SecretsControllerTest(ApiApplicationFactory factory)
         _secretRepository = _factory.GetService<ISecretRepository>();
     }
 
+    [Fact]
+    public async Task CreateSecret()
+    {
+        var organization = await SetupTest();
```

Comment:
Since we call setup in each test we can just use the constructor for this. https://xunit.net/docs/shared-context

Reply:
Resolved in https://github.com/bitwarden/server/pull/2292/commits/8141897b2ed8b0665184857fb54a9895f5d14b0e